File: php/v6.php
<?php

$user_ip = '::1';
$is_private = filter_var($user_ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6 | FILTER_FLAG_NO_PRIV_RANGE) === false ? 'false' : 'true';
$is_loopback = $user_ip === '::1' ? 'true' : 'false';

echo "Language: PHP\n";
echo "IP Address: $user_ip | Is Private: $is_private | Is Loopback: $is_loopback\n";
?>

File: golang/v6.go
package main

import (
	"fmt"
	"net/netip"
)

func main() {
	fmt.Println("Language: Go")
	ip := netip.MustParseAddr("::1")
	fmt.Printf("=== Checking IP: %s ===\n", ip)
	fmt.Printf("IsPrivate: %v\n", ip.IsPrivate())
	fmt.Printf("IsLoopback: %v\n", ip.IsLoopback())
	fmt.Println()
}
